jekyll/_cci2/workflows.md

```diff
@@ -300,62 +300,59 @@ workflows:
 
 In the example, `filters` is set with the `branches` key and the `only` key with the branch name. Any branches that match the value of `only` will run the job. Branches matching the value of `ignore` will not run the job. See the [Sample Sequential Workflow config with Branching](https://github.com/CircleCI-Public/circleci-demo-workflows/blob/sequential-branch-filter/.circleci/config.yml) for a full example.
 
-### Git Tag Job Execution
+### Using Tag Filters to Run Jobs
 
-CircleCI treats tag and branch filters differently when deciding whether a job should run.
+When running jobs,
+CircleCI treats tag and branch filters differently.
 
-1. For a branch push unaffected by any filters, CircleCI runs the job.
-2. For a tag push unaffected by any filters, CircleCI skips the job.
+1. If a job is unaffected by **branch** filters,
```

Comment:
This is still confusing. Let me chat with @lewang about it

Reply:
Whoops, did I make it worse? 😬